### PR TITLE
remote config to enable/disable location tracking for connections

### PR DESCRIPTION
**Pull request by anonymcek**
Use remote config for:
- enabling / disabling location tracking
- set accuracy of location if tracking is enabled

### CODE REVIEW

File: Covid/Services/LocationServices/BeaconManager.swift [hunk 1]
```diff
@@ -75,6 +76,12 @@ final class BeaconManager: NSObject {
             ]
         )
     }
+    
+    func activateLocationTrackingForBeacons() {
+        locationManager.requestAlwaysAuthorization()
```
Comment (bielikb) [inline Covid/Services/LocationServices/BeaconManager.swift:81]: out-of-the-scope koment:

vsimol som si, ze nehandlujeme niektore, edge casey, napr. ak OS killne appku a potom ju nasledne nastartatuje na zaklade location eventu, tento even nespracovavame..
po release sa na to mozme pozriet.

eg

**UIApplicationLaunchOptionsLocationKey**
_A key indicating that the app was launched to handle an incoming location event._

_The value of this key is an NSNumber object containing a Boolean value. You should use the presence of this key as a signal to create a CLLocationManager object and start location services again. Location data is delivered only to the location manager delegate and not using this key._

zdroj
https://developer.apple.com/documentation/uikit/uiapplicationlaunchoptionslocationkey

Comment (anonymcek) [reply Covid/Services/LocationServices/BeaconManager.swift:81]: dobbre vediet, teraz sa to snazime spravit pri akomkolvek spusteni :D

File: Covid/Services/LocationServices/BeaconManager.swift [hunk 2]
```diff
@@ -61,10 +62,10 @@ final class BeaconManager: NSObject {
     override private init() {
         super.init()
 
-        locationManager.requestAlwaysAuthorization()
-        locationManager.desiredAccuracy = kCLLocationAccuracyHundredMeters
-        locationManager.allowsBackgroundLocationUpdates = true
         locationManager.delegate = self
+        if (Firebase.remoteConfig?.configValue(forKey: "ibeaconLocationAccuracy").numberValue ?? -1) != -1 {
```
Comment (bielikb) [inline Covid/Services/LocationServices/BeaconManager.swift:66]: tento zapis budeme musiet v buducnosti vytunit :) 

idealne by sme zriesili eg enum pre keys, ktore registrujeme v remote konfigu, aby sme potom mohli pohodlne volat apinu ako napr.:

```
Firebase.value(for: . ibeaconLocationAccuracy).numberValue ...   
```

ci nieco podobne ;)

Comment (anonymcek) [reply Covid/Services/LocationServices/BeaconManager.swift:66]: presne nejaky enum, tieto keys, to uz len cakam kde sa pomylim, to je tak ked sa to snazis robit rychlo rychlo, mas to uz na 5 miestach a potom na kazde dalsie sa ti to nechce upravovat, ze by to bolo vela robboty a potom to mas zrazu na 30 miestach a uz sa ti to nechce vobec prepisovat, ale grcas ked to zapisujes 31. krat :)))

File: Covid/Services/LocationServices/BeaconManager.swift (hunk 1 above)
Comment (bielikb) [reply Covid/Services/LocationServices/BeaconManager.swift:81]: prave som mrkol, ako sme to riesili pred 5 rokmi v Downtowne, toto riesenieuz moze byt dost outdated, no mozno by nebolo od veci ho oskusat :)

```
    if (launchOptions[UIApplicationLaunchOptionsLocationKey]) {
        // [_locationManager startMonitoringSignificantLocationChanges]
        // [_beaconManager startMonitoringForRegion: ...]
    }
```

File: Covid/Services/LocationServices/BeaconManager.swift (hunk 2 above)
Comment (bielikb) [reply Covid/Services/LocationServices/BeaconManager.swift:66]: haha, presne, postupne skonvergujeme spat k 10 vyskytom ;)

**Review by bielikb — APPROVED** [review]

**Pull request merged**